python/cudf_polars/tests/expressions/test_rolling.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import pytest
+
+import polars as pl
+
+from cudf_polars import translate_ir
+
+
+def test_rolling():
+    dates = [
+        "2020-01-01 13:45:48",
+        "2020-01-01 16:42:13",
+        "2020-01-01 16:45:09",
+        "2020-01-02 18:12:48",
+        "2020-01-03 19:45:32",
+        "2020-01-08 23:16:43",
+    ]
+    df = (
+        pl.DataFrame({"dt": dates, "a": [3, 7, 5, 9, 2, 1]})
+        .with_columns(pl.col("dt").str.strptime(pl.Datetime))
+        .lazy()
+    )
+    q = df.with_columns(
+        sum_a=pl.sum("a").rolling(index_column="dt", period="2d"),
+        min_a=pl.min("a").rolling(index_column="dt", period="2d"),
+        max_a=pl.max("a").rolling(index_column="dt", period="2d"),
+    )
+    with pytest.raises(NotImplementedError):
+        _ = translate_ir(q._ldf.visit())
```

Comment:
This test seems a bit more complex than it needs to be. Won't a single rolling validate the same thing? Also, the grouped rolling test is much simpler below because it's not using datetimes; did you intentionally use datetimes to improve test coverage, or should we simplify this one to match the grouped test?

Reply:
This was just copied from a doctest in the polars docstring for rolling.